test/test-bug-ordered_pairs.lua

```diff
@@ -0,0 +1,19 @@
+-- Test based on real bug scenario -- https://github.com/lua-nucleo/lua-nucleo/issues/31
```

Comment:
This file should not be here. Please follow the way the rest of the tests are done.

BTW, if you would add a new test file (not sure if you should do that) do not forget to regenerate the list of tests.